src/Blazored.Modal/ModalAnimation.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Blazored.Modal
+{
+    public class ModalAnimation
```

Comment:
Apologies for not thinking of this before but what do you think to adding some static methods to make using the `ModalAnimation` class a bit easier? 

For example, when adding a global setting instead of doing this:

```razor
<BlazoredModal Animation="@(new ModalAnimation(ModalAnimationType.FadeIn, 2))"/>
```

It might be nicer to do this.

```razor
<BlazoredModal Animation="@ModalAnimation.FadeIn(2)"/>
```

This would require a static method on this class which would look like this.

```csharp
public class ModalAnimation
{
    public static ModalAnimation FadeIn(double duration)
    {
        return new ModalAnimation(ModalAnimationType.FadeIn, 2);
    }

    // Other code goes here...
}
```

What do you think?

Reply:
That actually sounds like a very good idea! Fixes some of the ugly stuff indeed.